cve_bin_tool/cvedb.py

```diff
@@ -53,7 +53,7 @@ class CVEDB:
     BACKUPCACHEDIR = DISK_LOCATION_BACKUP
     FEED = "https://nvd.nist.gov/vuln/data-feeds"
     LOGGER = LOGGER.getChild("CVEDB")
-    NVDCVE_FILENAME_TEMPLATE = "nvdcve-1.1-{}.json.gz"
+    NVDCVE_FILENAME_TEMPLATE = NVD_FILENAME_TEMPLATE
```

Comment:
I wonder if we should do a tiny refactor here so we're not storing it again in the class, but I can't decide if that actually makes any sort of meaningful difference other than making it easier to search on a single variable name.